src/intra_edge.rs

```diff
@@ -18,18 +18,22 @@ bitflags! {
 }
 
 impl EdgeFlags {
-    const LEFT_HAS_BOTTOM: Self = Self::union_all([
+    pub const LEFT_HAS_BOTTOM: Self = Self::union_all([
         Self::I444_LEFT_HAS_BOTTOM,
         Self::I422_LEFT_HAS_BOTTOM,
         Self::I420_LEFT_HAS_BOTTOM,
     ]);
 
-    const TOP_HAS_RIGHT: Self = Self::union_all([
+    pub const TOP_HAS_RIGHT: Self = Self::union_all([
```

Comment:
```suggestion
    pub const ALL_TOP_HAS_RIGHT: Self = Self::union_all([
```

Same reasoning and commit as above.